contracts/BandPriceFeed.sol

```diff
@@ -34,10 +39,17 @@ contract BandPriceFeed is IPriceFeed, BlockContext, CumulativeTwap {
     /// @dev anyone can help update it.
     function update() external {
         IStdReference.ReferenceData memory bandData = _getReferenceData();
-
         _update(bandData.rate, bandData.lastUpdatedBase);
     }
 
+    function cacheTwap(uint256 interval) external override returns (uint256) {
+        IStdReference.ReferenceData memory latestBandData = _getReferenceData();
+        if (interval == 0) {
+            return latestBandData.rate;
+        }
+        return _cacheTwap(interval, latestBandData.rate, latestBandData.lastUpdatedBase);
```

Comment:
why don't we return `_cacheTwap()` in function `getPrice()`

Reply:
Because `getPrice()` is a view function but `_cacheTwap()` is not.